lib/main_common.pm

```diff
@@ -1113,11 +1113,18 @@ sub load_console_server_tests {
         loadtest "console/php_pcre" if is_sle;
         # TODO test on SLE https://progress.opensuse.org/issues/31972
         loadtest "console/mariadb_odbc" if is_opensuse;
-        loadtest "console/php8" unless is_leap("<15.4") || is_sle("<15-SP4");
-        loadtest "console/php7";
-        loadtest "console/php7_mysql";
-        loadtest "console/php7_postgresql";
-        loadtest "console/php7_timezone";
+        if (is_leap("<15.4") || is_sle("<15-SP4")) {
+            loadtest "console/php7";
+            loadtest "console/php7_mysql";
+            loadtest "console/php7_postgresql";
+            loadtest "console/php7_timezone";
+        }
+        else {
+            loadtest "console/php8";
+            loadtest "console/php8_mysql";
+            loadtest "console/php8_postgresql";
+            loadtest "console/php8_timezone";
```

Comment:
This also will apply for Tumbleweed if I read this correctly? Can you give it a try too?...

Reply:
Yes it will. I've tried the tests "isolated" by now :)
http://10.161.229.196/tests/292

Reply:
lmk how a full run looks like

Reply:
All the full runs are running now. See PR description :)